### PR TITLE
Update SDK table to be consistent with other tables

### PR DESCRIPTION
**Pull request by zhenlan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Module | Platform | Sample | Release Notes
 ------ | -------- | ------ | -------------
 [Microsoft.Extensions.Configuration.AzureAppConfiguration](https://github.com/Azure/AppConfiguration-DotnetProvider)<br/>[![NuGet](https://img.shields.io/nuget/v/Microsoft.Extensions.Configuration.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | .Net Standard | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetCore) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md)
```
Comment (abhilasharora) [inline README.md:46]: `.Net Standard` -> `.NET Standard`

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Module | Platform | Sample | Release Notes
...
-[Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration](https://github.com/aspnet/MicrosoftConfigurationBuilders/tree/master/src/AzureAppConfig)<br/>[![NuGet](https://img.shields.io/nuget/v/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration/) | .NET Framework |  |
+[Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration](https://github.com/aspnet/MicrosoftConfigurationBuilders/tree/master/src/AzureAppConfig)<br/>[![NuGet](https://img.shields.io/nuget/v/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration/) | .NET Framework | [Sample](https://github.com/Azure/AppConfiguration/tree/master/examples/DotNetFramework/WebDemo) |
```
Comment (abhilasharora) [inline README.md:48]: This is the only table item with missing release notes. Is there a way we could reference or extract out the release notes for this package from the ones for the parent repository [here](https://github.com/aspnet/MicrosoftConfigurationBuilders/releases)?

File: README.md (hunk 1 above)
Comment (zhenlan) [reply README.md]: Updated all occurrences.

File: README.md (hunk 2 above)
Comment (zhenlan) [reply README.md]: Thanks. I added that link anyway. People have to eye-ball filter it :-)

**Pull request merged**